giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/config/PersistentContext.java

```diff
@@ -43,7 +41,7 @@ LocalContainerEntityManagerFactoryBean entityManagerFactory(DataSource dataSourc
         LocalContainerEntityManagerFactoryBean entityManagerFactoryBean = new LocalContainerEntityManagerFactoryBean();
         entityManagerFactoryBean.setDataSource(dataSource);
         entityManagerFactoryBean.setJpaVendorAdapter(new HibernateJpaVendorAdapter());
-        entityManagerFactoryBean.setPackagesToScan("edu.asu.diging.gilesecosystem.web.core.model", "edu.asu.diging.gilesecosystem.web.core.apps", "edu.asu.diging.gilesecosystem.web.core.users");
+        entityManagerFactoryBean.setPackagesToScan("edu.asu.diging.gilesecosystem.web.core.model", "edu.asu.diging.gilesecosystem.web.core.model.impl", "edu.asu.diging.gilesecosystem.web.core.apps", "edu.asu.diging.gilesecosystem.web.core.users", "edu.asu.diging.gilesecosystem.web.core.repository");
```

Comment:
I'm fairly positive that if `edu.asu.diging.gilesecosystem.web.core.model` is scanned, all subpackages are scanned too, so `edu.asu.diging.gilesecosystem.web.core.model.impl` is redudant.

Reply:
Also, does the repository folder contain entities? It shouldn't, so it should not need to be listed here.